### ISSUE TITLE
can you make it work with auto dark mode?

### ISSUE THREAD

**Issue by ghost** [rbika/easy-purple-incognito-theme#4]
can you make it work with auto dark mode?
when light switch to default light
when dark switch to default dark
always incognito purple

**Comment by rbika** [comment]
Hey, @yazan1107. I'll take a look but i don't think this is something available for custom themes. Do you know any custom theme that have this working?

**Comment by ghost** [comment]
> Hey, @yazan1107. I'll take a look but i don't think this is something available for custom themes. Do you know any custom theme that have this working?

not really, dark reader is not a theme but works per system theme(u can check it out)!

**Comment by rbika** [comment]
I took a look at this issue, but I'm afraid it is not currently possible to make this work with custom themes. Themes are very basic kind of extensions that do not allow this kind of control like dark reader. 😕

**Comment by ghost** [comment]
> I took a look at this issue, but I'm afraid it is not currently possible to make this work with custom themes. Themes are very basic kind of extensions that do not allow this kind of control like dark reader. 😕

is there an option to make incognito border theme only?
